test/unit/stellar_toml_resolver_test.js

```diff
@@ -184,5 +184,55 @@ FEDERATION_SERVER="https://api.stellar.org/federation"
             .then(() => tempServer.close());
         });
     });
+
+    it("rejects redirect response when allowedRedirects is not specified", function (done) {
+      if (typeof window != "undefined") {
+        return done();
+      }
+
+      let tempServer = http
+        .createServer((req, res) => {
+          res.writeHead(302, { location: "/redirect" });
+          return res.end();
+        })
+        .listen(4444, () => {
+          Resolver.resolve("localhost:4444", {
+            allowHttp: true,
+          })
+            .should.be.rejectedWith(/Maximum number of redirects exceeded/)
+            .notify(done)
+            .then(() => tempServer.close());
+        });
+    });
+
+    it("returns handled redirect when allowedRedirects is specified", function (done) {
```

Comment:
I don't see it being specified in the test below :thinking: :thinking:

Reply:
Yes silly mistake on my end, addressed it now